### PR TITLE
Cleaning up dollar vars

### PR DESCRIPTION
**Pull request by hkim823**

### CODE REVIEW

File: playbooks/roles/aws/templates/send-logs-to-s3.j2 [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #  This script can be called from logrotate
 #  to sync logs to s3
 
-if (( $EUID != 0 )); then
+if (( $EUID != 0 ));git  then
```
Comment (jarv) [inline playbooks/roles/aws/templates/send-logs-to-s3.j2]: typo?

**Comment by jarv** [comment]
I think I would remove these assignments altogether form the bash script if isntead it will be better to use the template vars:

```
S3_LOGFILE="{{ aws_s3_logfile }}"
NOTIFY_EMAIL={{ AWS_S3_LOGS_NOTIFY_EMAIL }}
FROM_EMAIL={{ AWS_S3_LOGS_FROM_EMAIL }}
AWS_CMD={{ aws_cmd }}
```

**Comment by feanil** [comment]
looks good, I agree with john, we should remove the variable assignments since they are no longer used anywhere.

**Comment by jarv** [comment]
given that there are still dollar variables in this template will it still throw warnings? @feanil I wasn't escaping these previously, are you surprised this ever worked?

**Comment by hkim823** [comment]
In my testing it seems to ignore $ variables if they aren't defined, it's only the defined ones it complains about.  

This is similar to how the update script in the edx_ansible is templated it and doesn't complain about that.  I just ran it on my sandbox looking for it to throw errors but it didn't seem to when I manually ran the aws play.

**Comment by hkim823** [comment]
I've updated a bunch, looking for another code review @feanil @jarv

**Comment by feanil** [comment]
looks good.

**Pull request merged**